resources-domain/resources-service/migrations/20230727084000-indexes-for-reindexing-query.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      CREATE INDEX IF NOT EXISTS "ResourceStringAttributes_resourceId_accountSid_idx"
```

Comment:
Frustrating that I had to create this - the primary key matches the same fields in the same order and similar PKs are used fort the same join in the query for the other attribute tables, but for some reason it insists on doing a full scan of this table without this secondary index in place.